home-exercises/src/io/codelex/arithmetic/practice/Exercise1Integers.java

```diff
@@ -0,0 +1,27 @@
+package io.codelex.arithmetic.practice;
+
+import java.util.Scanner;
+
+public class Exercise1Integers {
+    public static void main(String[] args) {
+        Scanner n1 = new Scanner(System.in);
+        System.out.println("Enter the first number: ");
+        int num1 = n1.nextInt();
+        Scanner n2 = new Scanner(System.in);
+        System.out.println("Enter the second number");
+        int num2 = n2.nextInt();
+        int diff1 = num1 - num2 ;
+        int diff2 = num2 - num1 ;
+        int sum = num1 + num2;
+
+        if (diff1 == 15) {
```

Comment:
Te vēl nedaudz pietrūkst - uzdevums bija, ka jāatgriež true, ja kāds no skaitļiem ir 15, vai to summa ir 15 vai starpība ir 15. Šeit redzu tikai summu un starpību.
Par starpību var izdarīt arī šādi - vienkārši atņem vienu skaitli no otra un pārbaudi vai absolūtā vērtība ir 15 un tas strādās neatkarīgi kurš skaitlis ir lielāks (metode `Math.abs()`)